src/hdtDefaultBBP.cpp

```
@@ -11,28 +17,110 @@ namespace hdt
 		return &s;
 	}
 
-	DefaultBBP::PhysicsFile_t DefaultBBP::scanBBP(RE::NiNode* scan)
+	std::optional<DefaultBBP::PhysicsFile_t> DefaultBBP::scanEmbeddedBBP(RE::NiNode* scan)
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)